manifests/params.pp

```diff
@@ -34,7 +34,9 @@
         $contrib_package_name = pick($contrib_package_name,'postgresql-contrib')
         $devel_package_name   = pick($devel_package_name, 'postgresql-devel')
         $java_package_name    = pick($java_package_name, 'postgresql-jdbc')
+        $docs_package_name    = pick($docs_package_name, 'postgresql-docs')
         $plperl_package_name  = pick($plperl_package_name, 'postgresql-plperl')
+        $plpython_pkg_name    = pick($plpython_pkg_name, 'postgresql-plpython')
```

Comment:
This should be `$plpython_package_name`. You will need to add spaces to all other parameter lines to keep alignment.